doc/revision_history.rst

```diff
@@ -9,12 +9,24 @@ Revision History
 v5.11.0
 =======
 
-Changes:
+Major changes:
```

Comment:
Are they really major changes? This seems like it would fit in a patch release.

Reply:
I believe it is necessary because I added a new function (assert_black_formatted) and added important features to CscCommander.